akka-stream/src/main/scala/akka/stream/javadsl/FileIO.scala

```diff
@@ -41,10 +45,10 @@ object FileIO {
    * set it for a given Source by using [[ActorAttributes]].
    *
    * @param f The file to write to
-   * @param append Whether or not the file should be overwritten or appended to
+   * @param options File open options
    */
-  def toFile(f: File, append: Boolean): javadsl.Sink[ByteString, CompletionStage[IOResult]] =
-    new Sink(scaladsl.FileIO.toFile(f, append).toCompletionStage())
+  def toFile(f: File, options: util.Set[StandardOpenOption] = new util.HashSet(util.Arrays.asList(WRITE, CREATE))): javadsl.Sink[ByteString, CompletionStage[IOResult]] =
```

Comment:
True. Just add it it to the method call above. (Which BTW needs an update to the ScalaDoc).